lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -355,35 +355,19 @@ function dosomething_global_get_user_language($user = NULL) {
  * @return array
  *  All of the supported countries
  */
- function dosomething_global_get_countries() {
-   $lang_map = variable_get('dosomething_global_language_map', '');
-   $countries = array();
-   foreach ($lang_map as $lang) {
-     $country = $lang['country'];
-
-     // Prevents global english from slipping through
-     if ($country != NULL) {
-       array_push($countries, $country);
-     }
-   }
-   array_push($countries, 'global');
-   return $countries;
- }
+function dosomething_global_get_countries() {
+  $lang_map = variable_get('dosomething_global_language_map', '');
+  $countries = array();
+  foreach ($lang_map as $lang) {
+    $country = $lang['country'];
 
-/**
- * Returns the session's country code, or `global` if it's not
- * one of our whitelisted global countries.
- *
- * @return string
- *   country code, or 'global'
- */
-function dosomething_global_get_country() {
-  $country_code = dosomething_settings_get_geo_country_code();
-  if (in_array($country_code, dosomething_global_get_countries())) {
-    return $country_code;
+    // Prevents global english from slipping through
+    if ($country != NULL) {
+      array_push($countries, $country);
+    }
   }
-
-  return 'global';
+  array_push($countries, 'global');
+  return $countries;
```

Comment:
hmm, the other place this is used is the [signup admin form](https://github.com/DoSomething/phoenix/blob/5896b2823f770c6aa3a1e9cf46d474ef4ae60564/lib/modules/dosomething/dosomething_signup/dosomething_signup.admin.inc#L22)

take a look locally, and in [stage](https://staging.beta.dosomething.org/admin/config/dosomething/opt_in) to see nothing changed